README.md

```diff
@@ -19,35 +19,70 @@ Python methods for numerical differentiation of noisy data, including multi-obje
         <img src="https://joss.theoj.org/papers/102257ee4b0142bf49bc18d7c810e9d5/status.svg"></a>
 </p>
 
-## Table of contents
```

Comment:
This takes up really valuable real estate at the top of the page, and the readme is short enough that the value of a TOC is unclear to me, so I nixed it.